server/metrics/metrics.go

```diff
@@ -227,6 +229,15 @@ func NewMetrics(info InstanceInfo) Metrics {
 	})
 	m.registry.MustRegister(m.httpErrorsTotal)
 
+	m.invalidTokenDetected = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemMSGraph,
+		Name:        "invalid_token_detected",
+		Help:        "The total number of times invalid token has been detected.",
```

Comment:
Naming suggestion: `_total` to match Promethes best practices for counters, but also proposing we embed the idea of `oAuth` and `invalidated` for clarity and distinction from other potential tokens. Thoughts?

```suggestion
		Name:        "invalid_oauth_token_invalidated_total",
		Help: "The total number of times an invalid oAuth token has been invalidated.",
```

Reply:
`invalid_oauth_token_invalidated_total` sounds like a lot of "invalidation", maybe `oauth_token_invalidated_total

Reply:
lets use this `invalid_oauth_token_invalidate_total` I going to change all the code around to make it consistent.

Reply:
Done